### ISSUE TITLE
What is "post-it note brainstorming"?

### ISSUE THREAD

**Issue by ddjepler** [grueproof/teamcharter#1]
For those unfamiliar with the process, an appendix, link, or similar would greatly help.  We muddled through and I think we did something close to what was intended, but it caught me off guard as an underprepared facilitator.

**Comment by grueproof** [comment]
Good point! I’ll add in a section for definition of terms and some links to videos were appropriate.
